.circleci/config.yml

```diff
@@ -186,23 +185,25 @@ jobs:
           name: Build Desktop Mac
           no_output_timeout: 45m
           environment:
-            CSC_LINK: resource/certificates/mac.p12
             USE_HARD_LINKS: 'false'
           command: |
             set +e
             source $HOME/.nvm/nvm.sh
             nvm use
 
+            pushd desktop
+
+            rbenv global $(cat .ruby-version) && rbenv rehash
```

Comment:
In previous iterations, I had `rbenv install` also. But to install new Rubies one also need to upgrade Homebrew, etc. 

Coupling the local Ruby version to be one supported by the CircleCI image seems like a decent tradeoff to avoid jumping through various hoops here.